packages/client-python/src/rocketride/core/runtime/manager.py

```
@@ -0,0 +1,251 @@
+        import aiohttp
+
+        # Normalize: ensure the URI has a scheme
+        if not uri.startswith(('http://', 'https://')):
+            uri = f'http://{uri}'
+
+        try:
+            async with aiohttp.ClientSession() as session:
+                async with session.get(
+                    uri,
+                    timeout=aiohttp.ClientTimeout(total=2),
+                ) as resp:
```

Ruff (0.15.7)

[warning] Use a single `with` statement with multiple contexts instead of nested `with` statements

Combine `with` statements

(SIM117)